testenvironments.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Where is the documentation on how `testenvironments.json` works?

Is there some way to put a comment in this file, or in `Terminal.sln` that helps people in the future understand what this is for, and why it's needed?

Reply:
Well it's a great feature of VS2022 to allow running and debugging unit tests on Windows and Linux. In the Test Explorer we choose on what environment we want to test.

Reply:
> Where is the documentation on how `testenvironments.json` works?

Here is some documentation https://learn.microsoft.com/en-us/visualstudio/test/remote-testing?view=vs-2022. It's still in experimental preview.

Here a screenshot of the VS2022 where are the test. Much more reliable and faster than VSCode .

![imagem](https://user-images.githubusercontent.com/13117724/196798350-5a6f94d3-b6cd-424e-b4e8-a9b507dc057a.png)